cypress/integration/channel/message_draft_with_attachment_then_switch_channel_sepc.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 4] */
+
+describe('Message Draft with attachment and Switch Channels', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M14126 Message Draft Pencil Icon - No text, only file attachment', () => {
+        // # Got to a test channel on the side bar
+        cy.get('#sidebarItem_town-square').scrollIntoView();
```

Comment:
Just do `cy.get('#sidebarItem_town-square').click({force: true})` cause we really just want to get to the channel.